practices/release_management.md

```diff
@@ -0,0 +1,123 @@
+- [Poor code quality](/problems/poor-code-quality)
+- [Increased cost](/problems/increased-cost)
+- [Demotivated Team](/problems/demotivated-team)
+- [Disconnect Between Business and IT](/problems/disconnect-between-business-and-it)
+- [Bad product-market fit](/problems/bad-product-market-fit)
+
+## How to Implement Release Management
+
+1. **Plan the release**
+   Planning stage requires a lot of time for structuring the release from start to finish. A strong release plan will help teams stay on track and meet expected standards, aims and requirements.
+
+2. **Build the release**
+   This stage involves the actual development (designing and building) of the product according to requirements defined during the release plan. After dealing with any issues that surfaced, the build must be subjected to real-world scenario testing. As the development team builds the product, it is tested for user acceptance by using a testing environment. This helps the team find bugs or issues that can occur in the real world. If any issues are identified, the build is sent back for development. The work flow becomes cyclic as the build is sent from the 'development' stage to 'testing' stage and vice versa until the release is approved.
+
+3. **User acceptance testing**
+   This involves the gathering of feedback from end users by allowing them to use an early version (free/beta/trial) of the product. This version is shared either online or with a large group of employees of a company. This is a vital step for the release management due to the large quantity of data gathered and fixes needed in order to prepare the product for its official launch.
+
+4. **Prepare the release**
+   This stage involves the final changes done to the product after considering everything learned from User Acceptance Testing phase. Release preparation also includes a final quality report by the Quality Assurance (QA) team. During this time the QA team will conduct final checks to see if the build meets the minimum standards of acceptability and bussiness requirements which was defined in the release plan. Once this is completed, the functional team will validate findings and work on finalizing the release for deployment. The approval of the product manager is needed before deploying the build into a live environment.
+
+5. **Deploy the release**
+   This is the final stage where the product is released to a live production environment. Apart from simply pushing out a build into production, this stage requires the need to educate the end users and your company. This can be achieved with the messaging, articles, documentation or demonstrations. This helps users to stay notified and updated on how to operate the new features of the release productively and efficiently. Finally, the development team should asess the performance of the release and discuss how to identify and document and issues to be addressed in the next iteration.
+
+## Common Pitfalls of Release Management
+
```

Comment:
We don't usually explain the pitfalls in our articles. Do you think you could rewrite them into single bullet point sentences?